CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 ### App Center Auth
 
+* **[Feature]** Expose the ID Token and Access Token JWTs in the `UserInformation` object passed to the sign in callback.
```

Comment:
"Token JWT" is redundant as the T stands for Token.
Also Android is not using a completion handler but returning a future, so it's not a callback per say.
Thus I suggest:

```suggestion
* **[Feature]** Expose the ID Token and Access Token (as raw JWT format) in the `UserInformation` object returned from the sign-in method.
```